### PR TITLE
database schemas

### PR DESCRIPTION
**Pull request by davidabrahams**

### CODE REVIEW

File: models/cohort.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var mongoose = require('mongoose');
+
+var cohortSchema = mongoose.Schema({
+  program: String,
```
Comment (davidabrahams) [inline models/cohort.js]: name

File: models/student.js [hunk 2]
```diff
@@ -3,9 +3,6 @@ var mongoose = require('mongoose');
 var studentSchema = mongoose.Schema({
   name: String,
   program: String,
```
Comment (davidabrahams) [inline models/student.js:5]: _cohort id

File: models/form.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+var mongoose = require('mongoose');
+
+var formSchema = mongoose.Schema({
+  studentID: String,
```
Comment (davidabrahams) [inline models/form.js]: _student_id

File: routes/index.js [hunk 4]
```diff
@@ -6,10 +6,9 @@ var passport = require('passport');
 var sendgrid = require('sendgrid')(process.env.SENDGRID_USERNAME, process.env.SENDGRID_PASSWORD);
 
 var User = require(path.join(__dirname, '../models/user'));
-var Attendance = require(path.join(__dirname, '../models/data')).attendance;
-var Entry = require(path.join(__dirname, '../models/data')).entry;
+var Form = require(path.join(__dirname, '../models/form'));
+var Cohort = require(path.join(__dirname, '../models/cohortEntry'));
```
Comment (nmohamed) [inline routes/index.js]: for consistency, just name it cohort instead of cohortentry

File: routes/index.js [hunk 5]
```diff
@@ -142,71 +141,186 @@ routes.POSTaddstudent = function(req, res, next) {
         newStudent: newStudent
       });
     })
-  })
+  }) var studentID = req.params._id;
```
Comment (shrutiyer) [inline routes/index.js]: Does this studentID have to be commented?

**Comment by shrutiyer** [comment]
:+1:

**Pull request merged**